resources/service-catalog-addons/charts/service-catalog-ui/templates/configmap.yaml

```diff
@@ -1,8 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Chart.Name }}-config
-  namespace: {{ .Release.Namespace }}
+  name: {{ .Chart.Name }}
```

Comment:
Maybe add nginx here like in add-ons-ui? https://github.com/asyncapi/asyncapi-react/blob/master/library/src/helpers/parser.ts#L45